pages/search.vue

```diff
@@ -0,0 +1,211 @@
+<script setup lang="ts">
+import VCard from '@nypublicradio/nypr-design-system-vue3/v2/src/components/VCard.vue'
+import VByline from '@nypublicradio/nypr-design-system-vue3/v2/src/components/VByline.vue'
+import useImageUrl from '~~/composables/useImageUrl'
+
+const route = useRoute()
+
+const querySlug = route.query.q
+const query = ref(querySlug || '')
+
+let articles = ref(
+  await searchArticlePages({ q: querySlug }).then(({ data }) =>
+    normalizeSearchArticlePagesResponse(data)
+  )
+)
+const articlesToShow = ref(10)
+
+const isSearching = ref(false)
+async function getSearchResults() {
+  isSearching.value = true
+  articles.value = await searchArticlePages({ q: query.value }).then(
+    ({ data }) => normalizeSearchArticlePagesResponse(data)
+  )
+  isSearching.value = false
+}
+
+onMounted(() => {
+  const { $analytics } = useNuxtApp()
+  $analytics.sendPageView({ page_type: 'search_page' })
+  getSearchResults()
+})
+
+const newsletterSubmitEvent = (e) => {
+  //emitted newsletter submit event, @Matt, not exactly sure how to get this work like you mentioned.
+  // sendEvent('click_tracking', {
+  //   event_category: 'Click Tracking',
+  //   component: 'Footer',
+  //   event_label: 'Become a member',
+  // })
+}
+</script>
+<template>
+  <div>
+    <div class="search-page">
+      <div class="pb-4">
+        <section>
+          <div class="content">
+            <div class="grid gutter-x-xl">
+              <div class="col-1 hidden xl:block"></div>
+              <div class="col">
```

Comment:
![image](https://user-images.githubusercontent.com/13803343/185167395-af77abd3-09a8-410a-9e5f-edf22eef6972.png)

Reply:
this uses the grid to lay out... and you can remove the max-width and width from ".search form"